modules/dhcp_native_ms/dhcp_native_ms_main.rb

```diff
@@ -10,11 +10,12 @@ class Provider < ::Proxy::DHCP::Server
     def initialize_for_testing(params)
       @name = params[:name] || @name
       @service = params[:service] || service
+      @subnets = params[:subnets] || @subnets
```

Comment:
Should this be `@managed_subnets` to match the instance var in the parent class?